lazy_tensor_core/lazy_tensor_core/core/extract_compiled_graph.py

```diff
@@ -0,0 +1,101 @@
+from lazy_tensor_core import _LAZYC
+import dataclasses
+from typing import List, Dict, Any, Callable
+import copy
+from torch import fx
+import torch
+
+debug = False
+
+@dataclasses.dataclass
+class GraphInputMatcher:
+    """
+    The GraphInputMatcher class setup the graph inputs for future calls after lazy tracing.
+    Specifically, those graph inputs corresponding to method parameters should be replaced with the
+    arguments for the current call.
+
+    tensor_id_to_arg_idx maps the tensor id to the parameter index.
+    graph_input_tensor_ids, graph_input_ivalues list the tensor_id and ivalue for each of the
+    TS/XLA graph inputs.
+    """
+    tensor_id_to_arg_idx: Dict[int, int]
+    graph_input_tensor_ids: List[int]
+    # there are 2 categories of graph_input_tensors.
+    # Category 1: those whose id are not found in tensor_id_to_arg_idx. These are
+    # most likely const tensors and we can get its content from graph_input_tensors
```

Comment:
what does this mean?
Does this mean these will be tensor constants in a JIT graph?

Reply:
Okay, I think it means scalars just looking through the rest of the code.
We should consider adding an example graph to illustrate the two categories.

Reply:
Yes, there are tensor constants. Examples can be found in test_exact_compuled_graph.py